lib/solidus_dev_support/templates/extension/bin/sandbox.tt

```diff
@@ -71,16 +75,17 @@ unbundled bundle exec rails generate solidus:install \
   --auto-accept \
   --user_class=Spree::User \
   --enforce_available_locales=true \
-  --with-authentication=false \
+  --with-authentication=<%= file_name != 'solidus_auth_devise' %> \
   --payment-method=none \
+  --frontend=${SOLIDUS_FRONTEND:-solidus_frontend} \
   $@
 
-unbundled bundle exec rails generate solidus:auth:install
-unbundled bundle exec rails generate ${extension_name}:install
+unbundled bundle exec rails generate solidus:auth:install --auto-run-migrations
+unbundled bundle exec rails generate ${extension_name}:install --auto-run-migrations
 
 echo
 echo "🚀 Sandbox app successfully created for $extension_name!"
-echo "🚀 Using $RAILSDB and Solidus $BRANCH"
-echo "🚀 Use 'export DB=[postgres|mysql|sqlite]' to control the DB adapter"
-echo "🚀 Use 'export SOLIDUS_BRANCH=<BRANCH-NAME>' to control the Solidus version"
-echo "🚀 This app is intended for test purposes."
+test -z "$DB"               && echo "- Use 'export DB=[postgres|mysql|sqlite]' to control the DB adapter"
+test -z "$SOLIDUS_BRANCH"   && echo "- Use 'export SOLIDUS_BRANCH=[master|v3.2|...]' to control the Solidus version"
+test -z "$SOLIDUS_FRONTEND" && echo "- Use 'export SOLIDUS_FRONTEND=[solidus_frontend|solidus_starter_frontend]' to control the Solidus frontend"
```

Comment:
Again, as it's for new extensions, maybe it's not worth it and we should just generate with solidus_starter_frontend. Thoughts?

Reply:
It's for supporting existing extensions rather than only creating new ones